src/legacy/core_plugins/expressions/index.ts

```diff
@@ -34,6 +34,7 @@ export default function DataExpressionsPlugin(kibana: any) {
     init: (server: Legacy.Server) => ({}),
     uiExports: {
       injectDefaultVars: () => ({}),
+      styleSheetPaths: resolve(__dirname, 'public/index.scss'),
```

Comment:
@streamich Inline styles are much harder to maintain. This particular component is now super delicate because it has to have just the right layout in order for the chart to render. Leaving it inline could make it brittle.

Reply:
@cchaos I'm not sure why you are saying it is much harder to maintain inline styles; in my experience they are much easier to maintain.

> Leaving it inline could make it brittle.

How would it make it brittle?

Reply:
Meaning that the specificity of inline styles is so high that if a consuming plugin passes down a class name to the main component, they'd have to use `!important`'s in order to get their styles applied. If this was a one-off component that is only used by a single component in a single place, inline styles would be fine. But this is being built so any application to use it therefore it needs to allow customization in way.

Reply:
Passing class names to React components is an anti-pattern. We should not be doing that in the first place. In React you usually want to re-use and share React components, not class names.

Customisation can be achieved through React props.

```tsx
<ExpressionRenderer />
<ExpressionRenderer small />
<ExpressionRenderer big />
<ExpressionRenderer flex />
<ExpressionRenderer somethingElse />
```

Reply:
I'm just trying to justify the usage of SASS and coming with the knowledge of consuming application needs for customizing styles as an author of EUI. In the end, it's on the current author who will need to maintain this plugin to decide how they want to handle these styles. But also don't forget about browser support. You don't get browser prefixing (for IE) with inline styles.

Reply:
> But also don't forget about browser support. You don't get browser prefixing (for IE) with inline styles.

I know, but I think we discussed it before, Flexbox does not need prefixing for IE11 (IE10 needs it, which we don't support).

Whether CSS is prefixed or not, it is still better to customise component through "semantic" props than "magic" CSS classes.